packages/core/src/config-engine/plugins.ts

```diff
@@ -216,16 +216,24 @@ export abstract class DmnoPlugin<
     initializedPluginInstanceNames.push(instanceName);
 
     // const callStack = new Error('').stack!.split('\n');
+    // console.log(callStack);
     // const pluginDefinitionPath = callStack[2]
     //   .replace(/.*\(/, '')
     //   .replace(/:.*\)/, '');
     // // special case for local dev when we have the plugins symlinked by pnpm
-    // if (pluginDefinitionPath.includes('/core/packages/plugins/')) {
-    //   const pluginPackageName
-    // } else {
+    // // if (pluginDefinitionPath.includes('/core/packages/plugins/')) {
+    // //   const pluginPackageName
+    // // } else {
 
-    // }
+    // // }
     // console.log(pluginDefinitionPath);
```

Comment:
can remove all this commented out code

Reply:
yeah I'll still do a big cleanup of this config engine stuff! still need to finish the nested branch resolver error stuff